api-server/rockcraft.yaml

```
@@ -80,7 +82,7 @@ parts:
 
       # Setup pip
       # TODO: Is this needed?
```

Let's add a small note.

```suggestion
      # This is required because the stage environment uses Python3.7. Do not update unless the upstream Dockerfile does.
```